app/controllers/secrets_controller.rb

```diff
@@ -66,8 +66,20 @@ def batch
       unless (secret = variable.last_secret)
         raise Exceptions::RecordNotFound, variable.resource_id
       end
-      
-      result[variable.resource_id] = secret.value
+
+      mime_type = variable.annotation('conjur/mime_type') || nil
```

Comment:
What is `|| nil` doing here?  Since only `false` and `nil` evaluate to falsey in ruby, the only thing this could be doing afaict is converting a `variable.annotation('conjur/mime_type')` of false to nil.  Is this in fact what it's intention is?  Otherwise you should be able to remove the `|| nil`.